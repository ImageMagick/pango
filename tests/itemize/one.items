<span font="Cantarell 11" fallback="false">a bâ€¨c</span>
