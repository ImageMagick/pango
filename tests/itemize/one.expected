<span font="Cantarell 11" fallback="false">a bâ€¨c</span>

Items:  a b            |[0x2028]       |c              
Font:   Cantarell 11   |Cantarell 11   |Cantarell 11   
Script: latin          |latin          |latin          
Lang:   en-us          |en-us          |en-us          
Bidi:   0              |0              |0              
Attrs:  [0,7]fallback=0|[0,7]fallback=0|[0,7]fallback=0
